src/dmd/target.h

```diff
@@ -47,16 +47,41 @@ enum class CPU
 
 struct TargetC
 {
+    typedef unsigned char CRuntime;
+    enum
+    {
+        CRuntime_Unspecified,
+        CRuntime_Bionic,
+        CRuntime_DigitalMars,
+        CRuntime_Glibc,
+        CRuntime_Microsoft,
+        CRuntime_Musl,
+        CRuntime_Newlib,
+        CRuntime_UClibc,
+        CRuntime_WASI,
+    };
```

Comment:
Use enum class, as C++11 is the baseline now, and these aren't used as bit flags.
```suggestion
    enum class Runtime : unsigned char
    {
        Unspecified,
        Bionic,
        DigitalMars,
        Glibc,
        Microsoft,
        Musl,
        Newlib,
        UClibc,
        WASI,
    };
```
Isn't this missing a flag such as `Platform` to distinguish between targets where the platform == C runtime (i.e OSX, Solaris, *BSD) and targets without a runtime?

Reply:
> Use enum class, as C++11 is the baseline now, and these aren't used as bit flags.

Good to know.

> Isn't this missing a flag such as Platform to distinguish between targets where the platform == C runtime (i.e OSX, Solaris, *BSD) and targets without a runtime?

I don't follow. This is explicitly a refactoring, I guess that can be added later if need be?